src/rpg_fixup.cpp

```diff
@@ -129,8 +168,107 @@ void RPG::SaveSystem::Fixup() {
 	}
 }
 
+void RPG::SaveSystem::UnFixup() {
+	const RPG::System& system = Data::system;
+
+	//TODO: Should be a C++14 polymorphic lambda
```

Comment:
what is a polymorphic lambda?

Reply:
A lambda where you can say `auto`. These 2 lambdas are identical except for the type of the parameter.